beacon-chain/execution/service.go

```diff
@@ -303,6 +310,19 @@ func (s *Service) updateConnectedETH1(state bool) {
 	s.updateBeaconNodeStats()
 }
 
+// IsExecutionClientSyncing returns a bool whether the execution client is syncing or not.
```

Comment:
Not much longer and a bit clearer
```suggestion
// IsExecutionClientSyncing returns true if the execution client is syncing, otherwise false.

```